src/Comcast/PhpLegalLicenses/Command/GenerateCommand.php

```diff
@@ -34,7 +34,7 @@ protected function configure()
      *
      * @return void
      */
-    protected function execute(InputInterface $input, OutputInterface $output)
+    protected function execute(InputInterface $input, OutputInterface $output): int
```

Comment:
Since some people are still using PHP 5.6 (and I list it as acceptable in `composer.json`) I'd rather leave off the `: int` return type hint.

Reply:
@orediggerco PR updated with the change.